lib/hal/led.h

```diff
@@ -0,0 +1,40 @@
+#ifndef HAL_LED_H
+#define HAL_LED_H
+
+#define LED_OFF    0x00
+#define LED_GREEN  0x01
+#define LED_RED    0x10
+#define LED_ORANGE 0x11
```

Comment:
nit: Can use an `enum`? Not sure what other maintainers think

Reply:
Yes, enum please.